src/main/js/controls/Graph/Graph.js

```diff
@@ -318,6 +319,9 @@ class Graph extends Construct {
         if (utils.notEmpty(content.config.values)) {
             removeNoDataView(this.svg);
         }
+        if (utils.notEmpty(content.config.values)) {
```

Comment:
Why is this duplicated?

Reply:
I have removed it I got added by mistake